### PR TITLE
chore: typo fixes

### PR DESCRIPTION
**Pull request by dundargoc**
This is where we collect all typo pull requests.

### CODE REVIEW

**Comment by zeertzjq** [comment]
This branch now has merge conflicts...

File: runtime/doc/diagnostic.txt [hunk 1]
```diff
@@ -373,39 +373,8 @@ config({opts}, {namespace})                          *vim.diagnostic.config()*
...
-                                     window as well as an (optional) highlight
-                                     group which will be used to highlight the
-                                     prefix. If {prefix} is a table, it is
-                                     interpreted as a [text, hl_group] tuple
-                                     as in |nvim_echo()|; otherwise, if
-                                     {prefix} is a string, it is prepended to
-                                     each diagnostic in the window with no
-                                     highlight.
-
+                                 • float: Options for floating windows. See
```
Comment (mjlbach) [inline runtime/doc/diagnostic.txt:376]: Is this correct/up-to-date? Seems like a big deletion

File: runtime/doc/lsp.txt [hunk 2]
```diff
@@ -873,6 +877,11 @@ start_client({config})                                *vim.lsp.start_client()*
                                            debounce occurs if nil
                                          • exit_timeout (number, default 500):
                                            Milliseconds to wait for server to
+                                           exit cleanly after sending the
```
Comment (mjlbach) [inline runtime/doc/lsp.txt:880]: Same, big change

File: runtime/lua/vim/diagnostic.lua [hunk 3]
```diff
@@ -570,33 +570,15 @@ end
...
----                            total) -> (string, string), where {i} is the index of the diagnostic
----                            being evaluated and {total} is the total number of diagnostics
----                            displayed in the window. The function should return a string which
----                            is prepended to each diagnostic in the window as well as an
----                            (optional) highlight group which will be used to highlight the
----                            prefix. If {prefix} is a table, it is interpreted as a [text,
----                            hl_group] tuple as in |nvim_echo()|; otherwise, if {prefix} is a
----                            string, it is prepended to each diagnostic in the window with no
----                            highlight.
+---       - float: Options for floating windows. See |vim.diagnostic.open_float()|.
```
Comment (mjlbach) [inline runtime/lua/vim/diagnostic.lua:573]: Same, big change.

File: runtime/lua/vim/diagnostic.lua [hunk 4]
```diff
@@ -1162,7 +1146,17 @@ end
 ---            - format: (function) A function that takes a diagnostic as input and returns a
 ---                      string. The return value is the text used to display the diagnostic.
 ---                      Overrides the setting from |vim.diagnostic.config()|.
----            - prefix: (function, string, or table) Prefix each diagnostic in the floating window.
+---            - prefix: (function, string, or table) Prefix each diagnostic in the floating
```
Comment (mjlbach) [inline runtime/lua/vim/diagnostic.lua:1149]: big change

File: runtime/doc/diagnostic.txt (hunk 1 above)
Comment (gpanders) [reply runtime/doc/diagnostic.txt:376]: See https://github.com/neovim/neovim/pull/16105

**Review by mjlbach — APPROVED** [review]
If we automate this would be nice to have the original PRs to see motivation (some are typos, some are doc changes)

**Pull request merged**